src/hooks/useChats.ts

```diff
@@ -0,0 +1,23 @@
+import { useFirestoreConnect } from "react-redux-firebase";
+
+import { venueChatsSelector } from "utils/selectors";
+
+import { useVenueId } from "hooks/useVenueId";
+import { useSelector } from "hooks/useSelector";
+
+export const useVenueChatConnect = () => {
+  const venueId = useVenueId();
+
+  useFirestoreConnect({
+    collection: "venues",
+    doc: venueId,
+    subcollections: [{ collection: "chats" }],
+    storeAs: "venueChats",
+  });
+};
```

Comment:
Can `venueId` ever be `undefined`/etc at this point? If so, we risk the same 'pathological query' we've seen in other places that would load all venues. It might be safer to use the following sort of pattern we've used in other places:
```suggestion
  useFirestoreConnect(venueId ? {
    collection: "venues",
    doc: venueId,
    subcollections: [{ collection: "chats" }],
    storeAs: "venueChats",
  } : undefined);
};
```

Reply:
fixed in 354167a550c42e93a27cc5c1e431145ff1caeef7